### PR TITLE
[flecs] update to 3.2.7

### PR DESCRIPTION
**Pull request by MUN1Z**
- [x] Changes comply with the [maintainer guide](https://github.com/microsoft/vcpkg-docs/blob/main/vcpkg/contributing/maintainer-guide.md)
- [x] SHA512s are updated for each updated download
- [ ] <strike>The "supports" clause reflects platforms that may be fixed by this new version</strike>
- [ ] <strike>Any fixed [CI baseline](https://github.com/microsoft/vcpkg/blob/master/scripts/ci.baseline.txt) entries are removed from that file.</strike>
- [ ] <strike>Any patches that are no longer applied are deleted from the port's directory.</strike>
- [x] The version database is fixed by rerunning `./vcpkg x-add-version --all` and committing the result.
- [x] Only one version is added to each modified port's versions file.

### CODE REVIEW

File: versions/f-/flecs.json [hunk 1]
```diff
@@ -1,5 +1,20 @@
...
+    {
+      "git-tree": "c32c42618c1867e847b276497cd12ce6c672d834",
+      "version": "3.2.6",
+      "port-version": 0
+    },
+    {
+      "git-tree": "74a7f74a2835d3946f05502ef44ba1d5d8b48eff",
+      "version": "3.2.5",
+      "port-version": 0
+    },
```
Comment (Adela0814) [inline versions/f-/flecs.json:17]: Please make sure only one version is added to modified port's versions file.

**Review by Adela0814 — CHANGES_REQUESTED** [review]

**Comment by Adela0814** [comment]
Note: I will be converting your PR to draft status. When you respond, please revert to "ready for review". That way, I can be aware that you've responded since you can't modify the tags.

File: versions/f-/flecs.json (hunk 1 above)
Comment (dg0yt) [inline versions/f-/flecs.json:17]:
```suggestion
```

**Comment by Adela0814** [comment]
Closing this PR since it seems that no progress is being made. Please ping us to reopen if work is still being done.

**Pull request closed without merging**